config/base.yaml

```diff
@@ -1,3 +1,7 @@
 log:
   stdout: true
   level: info
+
+dcClientConfig:
```

Comment:
Let's move it to `development.yaml`. 
Since this is good to have but still optional

Reply:
Done